test/read_mixin_test.rb

```diff
@@ -61,6 +61,7 @@
 
       describe "a single record" do
         before do
+          Post.unstub(:find_by_sql_without_kasket)
```

Comment:
Changes like this one in this test file were made because I discovered expectations weren't being checked ... they're unrelated to actual problem.